pkg/virt-handler/vm.go

```diff
@@ -3396,7 +3398,9 @@ func (d *VirtualMachineController) finalizeMigration(vmi *v1.VirtualMachineInsta
 		d.recorder.Event(vmi, k8sv1.EventTypeWarning, err.Error(), "failed to update guest memory")
 	}
 
-	if err := client.FinalizeVirtualMachineMigration(vmi); err != nil {
+	options := &cmdv1.VirtualMachineOptions{}
+	options.InterfaceMigration = domainspec.BindingMigrationByInterfaceName(vmi.Spec.Domain.Devices.Interfaces, d.clusterConfig.GetNetworkBindings())
```

Comment:
I am not sure where, but we need to keep only the pod network in the list.
Per what I noticed in the implementation, the migration method is applied on a interfaces although it is not relevant for non pod network ones.

It can be done in the virt-handler or in the virt-launcher. Depending on how this whole thing can evolve.

Reply:
I'm not sure that we should verify it is the pod network. Maybe there is some use case a secondary interface has to be reconnected.
The migration method is per plugin, no matter if the interface is the pod one or not.
If a user wants to use the same plugin for primary and secondary interfaces and want different behavior upon migration, he has to configure two plugins in the kubevirt CR.
What do you think?